openedx/core/djangoapps/staticfiles/apps.py

```diff
@@ -0,0 +1,33 @@
+"""
+Configuration for static assets. Shared by LMS and CMS.
+"""
+from django.contrib.staticfiles.apps import StaticFilesConfig
+
+
+class EdxPlatformStaticFilesConfig(StaticFilesConfig):
+    """
+    A thin wrapper around the standard django.contrib.staticfiles app
+    which adds the proper file & folder ignore patterns for edx-platform
+    static asset collection.
+
+    This allows devs & operators to run:
+        ./manage.py [lms|cms] collectstatic
+
+    instead of:
+        ./manage.py [lms|cms] collectstatic --ignore geoip --ignore sass ... etc.
+    """
+    name = 'openedx.core.djangoapps.staticfiles'
```

Comment:
I'm not sure but I think the failed tests are due to this change, please consider to keep the original name since changing it can cause problems in other parts of the code that depend on it.

Reply:
@andrey-canon I did end up reverting this line.

I believe tests were failing because putting `'openedx.core.djangoapps.staticfiles'` in INSTALLED_APPS wasn't enough. I needed to specify the full path to the config class: `'openedx.core.djangoapps.staticfiles.apps.EdxPlatformStaticFilesConfig'`.

I believe that's because the class isn't a direct subclass of `AppConfig`, so Django won't consider it the "default" app config class for our new staticfiles app.